CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+- Added links to supported features tables for 3rp party operating
```

Comment:
Changelog entries are meant to be copied to the packages `.changes` file literally.  Thus we must avoid abbreviations.